ManageWikiSettings.php

```diff
@@ -349,6 +349,23 @@
 		'section' => 'edit',
 		'help' => 'Show more authors.',
 	],
+	'wgRestrictionLevels' => [
+		'name' => 'Restriction Levels',
+		'from' => 'mediawiki',
+		'restricted' => false,
+		'type' => 'userrights',
+		'options' => [
+			'no protection' => '',
```

Comment:
Need to provide options here. The below part only makes sure it's the default.

Reply:
Add all the options that wikis are using see https://github.com/miraheze/mw-config/blob/1e327aade41e4a316a6fda8394e243696e232581/LocalSettings.php#L2867